packages/reactDom/test/test_RSC_model.ml

```diff
@@ -334,32 +334,6 @@ let client_with_element_props () =
       "0:\"$1\"\n";
     ]
 
-let client_with_promise_props () =
-  let app () =
-    React.Upper_case_component
-      (fun () ->
-        React.list
-          [
-            React.createElement "div" [] [ React.string "Server Content" ];
-            React.Client_component
-              {
-                props =
-                  [ ("promise", React.Promise (delayed_value ~ms:200 "||| Resolved |||", fun res -> `String res)) ];
-                client = React.string "Client with Props";
-                import_module = "./client-with-props.js";
-                import_name = "ClientWithProps";
-              };
-          ])
-  in
-  let%lwt stream = ReactServerDOM.render_model (app ()) in
-  assert_stream stream
-    [
-      "2:I[\"./client-with-props.js\",[],\"ClientWithProps\"]\n";
-      "1:[[\"$\",\"div\",null,{\"children\":\"Server Content\"}],[\"$\",\"$2\",null,{\"promise\":\"$@3\"}]]\n";
-      "0:\"$1\"\n";
-      "3:\"||| Resolved |||\"\n";
-    ]
-
```

Comment:
Why delete this?

Reply:
It was deleted by mistake while cleaning the code. Thx